packages/cms-cli/commands/functions/build.js

```diff
@@ -0,0 +1,73 @@
+const ora = require('ora');
+const {
+  addPortalOptions,
+  addConfigOptions,
+  setLogLevel,
+  getPortalId,
+  addUseEnvironmentOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cms-lib');
+const { logger } = require('@hubspot/cms-lib/logger');
+const { buildPackage } = require('@hubspot/cms-lib/api/functions');
+const { validatePortal } = require('../../lib/validation');
+
+const makeSpinner = (functionPath, portalIdentifier) => {
+  return ora(
+    `Building new bundle for '${functionPath}' on portal '${portalIdentifier}'.\n`
+  );
+};
+
+const loadAndValidateOptions = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath, options);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validatePortal(options)))) {
+    process.exit(1);
+  }
+};
+
+exports.command = 'build <path>';
+exports.describe =
+  'builds a new dependency bundle for the specified function folder';
+
+exports.handler = async options => {
+  loadAndValidateOptions(options);
+
+  const { path: functionPath } = options;
+  const portalId = getPortalId(options);
+  const spinner = makeSpinner(functionPath, portalId);
+
+  trackCommandUsage('functions-build', { functionPath }, portalId);
+
+  logger.debug(`Starting build for function(s) with path: ${functionPath}`);
+
+  spinner.start();
+  await buildPackage(portalId, `${functionPath}/package.json`);
+  spinner.stop();
+  logger.success(
```

Comment:
I know I've asked this before, but is there any chance we can output a direct link to the invocation of the function?

Reply:
Not until we have the `content` scope group available to the personal-access-key -- I think Allison is working on this(can't find her public github).

Reply:
I can get back on that PR, if it's blocking you. Completely slipped my mind!

Reply:
I think it would be a good improvement. Once the PAK has `content` we can retrieve the website `origin` to generate a link that executes the individual functions(`GET`s of course). We have the `/_hcms/api/endpointName` part accessible already through the response that `hs list` gets, and with the `origin` we will have a full URL.